templates/complete_saml_extension_body.html

```diff
@@ -0,0 +1,30 @@
+{{define "complete_saml_extension_body"}}
+
+<html>
+    <head>
+        <script>
+            document.addEventListener("DOMContentLoaded", function(event) {
+                var extensionId = {{.ExtensionId}};
+
+                if (!extensionId) {
+                    return;
+                }
+
+                chrome.runtime.sendMessage(
```

Comment:
see my comments above

Reply:
This would not be a concern as it's sending a message to a specific extension (based on extensionId)